velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +92,17 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct InverseNormalCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(double& result, double m, double sd, double p) {
+    VELOX_USER_CHECK((p >= 0) && (p <= 1), "p must be 0 > p > 1");
+    VELOX_USER_CHECK_GT(sd, 0, "standardDeviation must be > 0");
+
+    result = m + sd * 1.4142135623730951 * boost::math::erf_inv(2 * p - 1);
```

Comment:
1.4.14 is square root of 2. Please add a constant variable for it, and use it in the formula.